redis/Module.md

```diff
@@ -0,0 +1,102 @@
+## Overview
+
+Ballerina connector for Redis is connecting Redis via Ballerina language easily. This module provides the functionality required to access and manipulate key-value data stored in a Redis database this module provides API functions for manipulating Redis String, Hash, List, Set and Sorted Set data types.
```

Comment:
Suggestion:
The Ballerina connector for Redis allows you to connect to Redis and manipulate key-value data stored in a Redis database. You can use the connector to manipulate string, hash, list, set, and sorted set data types. The functionality of the Ballerina connector for Redis is limited to the use of Redis as a database and cache. The message broker functionality of Redis is not yet exposed through this module.